tools/codegen-crds/README.md

```diff
@@ -0,0 +1,81 @@
+# Codegen-crds
+
+This tool simplifies the process of generating CRD schemas across a large number of API groups and versions.
+It's aim is to encode specific knowledge about the schemapatch generators we use and the yaml patch methods we include
+in this repo and handle the complexity around running those.
+
+## Usage
+
+The tool can be compiled in the normal way using either `go build` or `make codegen-crds` from the root of the tools
+module.
+When using make the tool will output to `$(TOOLS_MODULE)/_output/bin/$(GOOS)/$(GOARCH)/bin/codeged-crds`.
+
+The function has the following arguments:
+- `--api-group-versions` - A comma separated list of group versions to generate, e.g. apps/v1,machine/v1,machine/v1beta1.
```

Comment:
those are the packages after the basedir, not the actual group-versions, right?  Want to keep the arg name or change it?

Reply:
Everywhere else in our tooling we refer to these as `api-group-versions`, they just aren't the fully qualified group names. Would your suggestions be `packages` or `api-packages`? I could see that.

The one thing I don't want to do though is make this so that it requires GOPATH (and calling them packages could become confusing if people start putting `gihub.com/openshift/api/package/version`, a lot of our tooling has required this in the past and it makes it difficult for folks when the tools suddenly start dumping files in random locations because they do/don't have GOPATH set. The current approach makes this consistent across both